src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Refresh.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using Microsoft.PowerFx.Core.App.ErrorContainers;
+using Microsoft.PowerFx.Core.Binding;
+using Microsoft.PowerFx.Core.Binding.BindInfo;
+using Microsoft.PowerFx.Core.Entities;
+using Microsoft.PowerFx.Core.Errors;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.Functions.Publish;
+using Microsoft.PowerFx.Core.Localization;
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Syntax;
+
+namespace Microsoft.PowerFx.Core.Texl.Builtins
+{
+    // Refresh(source:*[...]) : b
+    // Refresh(source:![...]) : b
+    internal sealed class RefreshFunction : BuiltinFunction
```

Comment:
>RefreshFunction [](http://example.com/codeflow?start=26&length=15)

I'm assuming this is straight from PA. Any changes?

Reply:
Correct.
Only line 94 with the new interface